### PR TITLE
repair check readme if not urls into readme

### PR DESCRIPTION
**Pull request by kaadevelop**

### CODE REVIEW

File: pylint_odoo/checkers/itp_checks.py [hunk 1]
```diff
@@ -185,15 +185,17 @@ def _check_readme_versions(self):
...
+                for url in urls:
+                    odoo_version = re.search('/(\d+\.\d)(/|$)', url)
+                    if odoo_version:
+                        if valid_odoo_version != odoo_version.group(1):
+                            print('Odoo version in link %s is not valid' % (url))
+                            return False
+                    else:
+                        return True
+            else:
+                return True
```
Comment (yelizariev) [inline pylint_odoo/checkers/itp_checks.py]: You don't need to change indent

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pylint_odoo/checkers/itp_checks.py [hunk 2]
```diff
@@ -185,15 +185,17 @@ def _check_readme_versions(self):
...
-                    return True
+            if urls != []:
+                for url in urls:
+                    odoo_version = re.search('/(\d+\.\d)(/|$)', url)
+                    if odoo_version:
+                        if valid_odoo_version != odoo_version.group(1):
+                            print('Odoo version in link %s is not valid' % (url))
+                            return False
+                    else:
+                        return True
```
Comment (yelizariev) [inline pylint_odoo/checkers/itp_checks.py]: wrong

Comment (kaadevelop) [reply pylint_odoo/checkers/itp_checks.py]: tramp xDDDD

File: pylint_odoo/checkers/itp_checks.py [hunk 3]
```diff
@@ -181,19 +181,21 @@ def _check_readme_versions(self):
...
+            urls.extend(url)
+        for url in urls:
+            odoo_version = re.search('/(\d+\.\d)(/|$)', url)
+            if not odoo_version:
+                continue
+            if valid_odoo_version != odoo_version.group(1):
+                print('Odoo version in link %s is not valid' % (url))
+                return False
+        else:
+            return True
```
Comment (yelizariev) [inline pylint_odoo/checkers/itp_checks.py]: It's the same as without else. Just return True

**Pull request merged**